build_tools/kokoro/gcp_ubuntu/cmake/linux/riscv64/abs_test.txt

```diff
@@ -0,0 +1,3 @@
+// RUN: ${TEST_MODULE_CMD} --module_file="${BUILD_RISCV_DIR}/iree-run-module-llvm_aot.vmfb" \
```

Comment:
The main feature I want to introduce is "XFAIL", which would allow us to catch the unexpected pass/fail. We currently have a failing test `person_detect_rvv_test` that's pending upstream integration fix. In the future we want to catch the breakage (not necessarily fixable within IREE framework) during integration.